package.json

```diff
@@ -12,14 +12,16 @@
     "clean": "node scripts/clean.js",
     "build:ts": "tsc",
     "build:gitignore": "node scripts/update_gitignore_for_tsc_output.js run",
-    "format": "prettier --write \"{lib,bin,scripts,test}/**/*.js\"",
-    "format:check-difference": "prettier --list-different \"{lib,bin,scripts,test}/**/*.js\"",
+    "format": "prettier --write' --list-different .",
+    "format:check": "prettier --list-different .",
     "debug:test": "mocha --inspect-brk --exit -t 0 test/all-tests-suite.js",
     "debug:tape": "node --inspect-brk test/tape/index.js",
     "coveralls": "nyc report --reporter=lcov",
-    "lint": "eslint \"lib/**/*.js\" \"test/**/*.js\" \"bin/**/*.js\"",
+    "lint": "eslint --cache --fix '**/*.js'",
```

Comment:
lint by definition is about checking, it would be better to have explicit `lint:fix` command for fixing.
And CI shouldn't be running it.

Reply:
Sure. But you'll end up with two different patterns for similar things. For ESLint you'll need ":fix" to make changes but not for Prettier. And for Prettier you'll need ":check" for check-only but not for ESLint.

Given the formatter and linter will always both run (in check-mode for CI and fix-mode locally) does it not make more sense to a)  follow the same script naming pattern; and b) be joined the same command ?